lib/shopify_api/cache_supervisor.ex

```diff
@@ -0,0 +1,35 @@
+defmodule ShopifyAPI.CacheSupervisor do
+  @moduledoc """
+  The CacheSupervisor provides a convient way to monitor, control, and start all the ShopifyAPI cache servers.
```

Comment:
Typo in `convenient`. Would likely be good to pull the "rationale" bit up to the top as well, explaining that this is offered as a standalone supervisor to allow for startup customization.